# HG changeset patch
# Parent  12c3d129c52f8251dc9bb2735dabd80dd0d89d82
1c erp версия 19

diff --git a/bge/oxmloutx.p b/bge/oxmloutx.p
--- a/bge/oxmloutx.p
+++ b/bge/oxmloutx.p
@@ -861,7 +861,7 @@ on error undo, return error return-value
         
         sw:start-element ("GC-ERPRN") .
         
-        sw:insert-attribute ("xmlns", "http://www.rosneft.ru/GasComplex/Retail/18.0") .
+        sw:insert-attribute ("xmlns", "http://www.rosneft.ru/GasComplex/Retail/19.0") .
         sw:insert-attribute ("xmlns:xs", "http://www.w3.org/2001/XMLSchema") .
         sw:insert-attribute ("xmlns:xsi", "http://www.w3.org/2001/XMLSchema-instance") .
 
