# HG changeset patch
# Parent  551cad110632cb26e25a12151588dde557c79114
1c erp версия 13

diff --git a/bge/oxmloutx.p b/bge/oxmloutx.p
--- a/bge/oxmloutx.p
+++ b/bge/oxmloutx.p
@@ -861,7 +861,7 @@ on error undo, return error return-value
         
         sw:start-element ("GC-ERPRN") .
         
-        sw:insert-attribute ("xmlns", "http://www.rosneft.ru/GasComplex/Retail/12.0") .
+        sw:insert-attribute ("xmlns", "http://www.rosneft.ru/GasComplex/Retail/13.0") .
         sw:insert-attribute ("xmlns:xs", "http://www.w3.org/2001/XMLSchema") .
         sw:insert-attribute ("xmlns:xsi", "http://www.w3.org/2001/XMLSchema-instance") .
 
